src/Microsoft.AspNetCore.Components.Browser.JS/Microsoft.AspNetCore.Components.Browser.JS.csproj

```diff
@@ -12,16 +12,12 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <ProjectReference Include="..\..\blazor\src\Microsoft.AspNetCore.Blazor.BuildTools\Microsoft.AspNetCore.Blazor.BuildTools.csproj" PrivateAssets="all" ReferenceOutputAssembly="false" />
     <PackageReference Include="Microsoft.Extensions.FileProviders.Embedded" Version="$(AspNetCorePackageVersion)" />
     <WebpackInputs Include="**\*.ts" Exclude="node_modules\**" />
     <WebpackInputs Include="..\..\modules\jsinterop\src\Microsoft.JSInterop.JS\src\**" />
   </ItemGroup>
 
-  <Import Project="..\..\blazor\src\Microsoft.AspNetCore.Blazor.BuildTools\BuildToolsExe.props" />
-
   <Target Name="EnsureNpmRestored" Condition="!Exists('node_modules')">
-    <Exec Command="$(BlazorBuildToolsExe) checknodejs -v 8.3.0" />
```

Comment:
I think we can just live without this. It only affects people building from source, not people consuming the shipped packages, and it only affects the case where the build was going to fail anyway because you don't have Node on `PATH`.